### PR TITLE
feat: adding stacking guide

### PR DESCRIPTION
**Pull request by agraebe**
fixing #637

### CODE REVIEW

File: src/common/navigation.yaml [hunk 1]
```diff
@@ -56,8 +57,9 @@ sections:
         sections:
           - title: Tutorials
             pages:
+              - path: /integrate-stacking
               - path: /install-api
-              - path: /installing-memcached
+              - path: /install-memcached
```
Comment (aulneau) [inline src/common/navigation.yaml:62]: We have to be very careful with renaming paths. We need to create a redirect anytime we do for a page that has been active.

Comment (agraebe) [reply src/common/navigation.yaml:62]: great point, adding a redirect!

File: src/pages/stacks-blockchain/stacking.md [hunk 2]
```diff
@@ -1,50 +1,54 @@
...
+Stacks token holders do not automatically receive Stacking rewards. Instead, they must:
 
-### Integrating stacking
+- Commit to participation before a reward cycle begins
+- Hold ~94.000 Stacks tokens
+- Lock up Stacks tokens for a specified period
+- Set a Bitcoin address to receive rewards
 
-Ready to integrate stacking in your wallet or exchange?
+Token holders will have to use a wallet or exchange that supports participation in Stacking.
```
Comment (psq) [inline src/pages/stacks-blockchain/stacking.md:37]: this is true, but not limited to these 2.  Stacking is done by calling a contract, so anything able to call this contract would also qualify.

File: src/pages/stacks-blockchain/stacking.md [hunk 3]
```diff
@@ -1,50 +1,54 @@
...
+- A reward cycle consists of two phases: prepare and reward
+- During the prepare phase, miners decide on an anchor block and a reward set. Mining any descendant forks of the anchor block requires transferring mining funds to the appropriate reward addresses. The reward set is the set of Bitcoin addresses which will receive funds in the reward cycle
+- Miners register as leader candidates for a future election by sending a key transaction that both burns cryptocurrency (proof-of-burn) and spends energy (proof-of-work). The transaction also registers the leader's preferred chain tip (must be a descendant of the anchor block) and commitment of funds to 5 addresses from the reward set
+- Token holders register for the next rewards cycle by broadcasting a signed message that locks up associated Stacks tokens for a protocol-specified lockup period, specifies a Bitcoin address to receive the funds, and votes on a Stacks chain tip
+- The leader that commits to the winning chain tip and the peers who also burn for that leader collectively share the reward, proportional to how much each one burned
```
Comment (psq) [inline src/pages/stacks-blockchain/stacking.md:53]: multiple leaders can commit to the same chain tip (hopefully, they all do, or this means a fork), it is the leader that wins the sortition, and whoever supported him that will get the reward.
That leader either transfers (or burns) btc, depending on several factors.

File: src/pages/stacks-blockchain/stacking.md [hunk 4]
```diff
@@ -1,50 +1,54 @@
...
+
+-> Fun fact: The Stacking consensus algorithm is implemented as a smart contract, using [Clarity](/smart-contracts/overview).
+
+- Progression in Stacking consensus happens over reward cycles (with a fixed length). In each reward cycle, a set of Bitcoin addresses are iterated over
+- A reward cycle consists of two phases: prepare and reward
+- During the prepare phase, miners decide on an anchor block and a reward set. Mining any descendant forks of the anchor block requires transferring mining funds to the appropriate reward addresses. The reward set is the set of Bitcoin addresses which will receive funds in the reward cycle
+- Miners register as leader candidates for a future election by sending a key transaction that both burns cryptocurrency (proof-of-burn) and spends energy (proof-of-work). The transaction also registers the leader's preferred chain tip (must be a descendant of the anchor block) and commitment of funds to 5 addresses from the reward set
```
Comment (psq) [inline src/pages/stacks-blockchain/stacking.md:51]: the key transaction is not a burn.  That amount is recoverable after a commit used the registered key (or when a key can be reused multiple times, after a miner no longer wishes to use that key)

Comment (psq) [reply src/pages/stacks-blockchain/stacking.md:51]: the commit transaction is what registers the chaintip the miner is going to mine if it wins

File: src/pages/stacks-blockchain/stacking.md [hunk 5]
```diff
@@ -1,50 +1,54 @@
 ---
 title: Understand Stacking
-description: Introduction to the consensus algorithm of the Stacks 2.0 blockchain
+description: Introduction to the rewarding mechanism of Proof-of-Transfer
```
Comment (psq) [inline src/pages/stacks-blockchain/stacking.md:3]: "reward" might be a better choice here.

File: src/pages/stacks-blockchain/stacking.md (hunk 2 above)
Comment (agraebe) [reply src/pages/stacks-blockchain/stacking.md:37]: yeah, good point. at this point in the flow, the reader doesn't know yet about stacking details like that it is just a contract. let me see how to be clear but also not overwhelming

File: src/pages/stacks-blockchain/stacking.md (hunk 3 above)
Comment (agraebe) [reply src/pages/stacks-blockchain/stacking.md:53]: thanks!

File: src/pages/stacks-blockchain/stacking.md (hunk 4 above)
Comment (agraebe) [reply src/pages/stacks-blockchain/stacking.md:51]: how would you rephrase the above to cover that? I'd keep in mind that this is an intro to the topic, so rather stay high level and simplify where possible

File: src/pages/stacks-blockchain/stacking.md (hunk 5 above)
Comment (agraebe) [reply src/pages/stacks-blockchain/stacking.md:3]: 👍

**Comment by agraebe** [comment]
closing in favor of #805

**Pull request closed without merging**